[Bug][Sonar] compile failed

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.


### What happened

`make all` exited with error
```
#17 140.5 2023/05/23 08:59:28 ParseComment error in file /app/backend/plugins/sonarqube/api/proxy.go :can not parse router comment "/plugins/sonarqube/connections/{connectionId}/proxy/rest/{*path} [GET]"
#17 140.5 make[1]: *** [Makefile:38: swag] Error 1
#17 140.5 make[1]: Leaving directory '/app/backend'
#17 140.5 make: *** [Makefile:81: build-server] Error 2
------
ERROR: failed to solve: executor failed running [/bin/sh -c export CGO_ENABLED=1 && make all]: exit code: 2
```

### What do you expect to happen

Should compile successfully.

### How to reproduce

1. Clone the repository
2. Checkout to `release-v0.17` branch
3. Run `export CGO_ENABLED=1 && make all`
4. Wait a moment and it reports error.

### Anything else

_No response_

### Version

release-v0.17

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/backend/plugins/sonarqube/api/proxy.go
+++ b/backend/plugins/sonarqube/api/proxy.go
@@ -36,7 +36,7 @@ import (
 // @Success 200  {object} interface{} "Success"
 // @Failure 400  {string} errcode.Error "Bad Request"
 // @Failure 500  {string} errcode.Error "Internal Error"
-// @Router /plugins/sonarqube/connections/{connectionId}/proxy/rest/{*path} [GET]
+// @Router /plugins/sonarqube/connections/{connectionId}/proxy/rest/{path} [GET]
 func Proxy(input *plugin.ApiResourceInput) (*plugin.ApiResourceOutput, errors.Error) {
 	connection := &models.SonarqubeConnection{}
 	err := connectionHelper.First(connection, input.Params)

